src/loki/reach.cc

```diff
@@ -67,7 +67,9 @@ directed_reach Reach::operator()(const DirectedEdge* edge,
   // and we can expand from the start of a complex restriction because only the end would mark a
   // potential stopping point (maybe a path followed the restriction)
 
-  // seed the expansion with a place to start expanding from
+  // seed the expansion with a place to start expanding from, set the number of labels we want to use
+  // we're finding nodes here so we'll double it assuming we queue less edges than nodes we see
+  max_reserved_labels_count_ = max_reach * 2;
```

Comment:
did you find it experimentally ?

Reply:
To be honest, I don't quite understand why this is. Kevin seemed to be doing it.
This is related to this issue https://github.com/valhalla/valhalla/issues/2808

Reply:
ok. I think rough theoretical result can be obtained from here https://en.wikipedia.org/wiki/Planar_graph

Reply:
it was just an attempt to right size the labels allocation for the reach computation. as the comment says, since the reach is based on nodes reached, i couldnt use max_reach directly to guess at how many edges i would ahve to see to hit the node reach threshold so i guessed that i'd see about 2 edges for every new node that i found. we could arrive at this empirically if we wanted to